ddcz/templates/base/right_menu.html

```diff
@@ -0,0 +1,70 @@
+<ul class="nav_border">
+    <li><span class="list_heading">Vzhled</span>
+        <ul class="list_box">
+            <li>Family Silver</li>
+            <li>Historic</li>
+            <li><a href="{% url 'ddcz:change-skin' %}?skin=light&redirect={{ request.get_full_path }}">Light</a></li>
```

Comment:
I think we shouldn't be using request in the templates. I propose introducing new variable in the common variables middleware, I'd call it `current_page_url`